### ISSUE TITLE
Change to focus mode or always allow click event when accessing a link / a tag

### ISSUE THREAD

**Issue by Anubarak** [nvaccess/nvda#12496]
### Is your feature request related to a problem? Please describe.

I often have the case when creating websites that NVDA does not switch into focus mode when the focus is on a link. To be honest I cannot think of a single use case that would be suitable to disable the `Enter` key when I focus a link.
I `tab` through the different inputs, end on a link to another page, press enter and expect something to happen - but nothing happens... I'm not entirely sure if it's my fault as a developer or if my link is wrong in some way but I was not able to find a way to force NVDA to focus mode in order to access links. I know I can press `NVDA` + `SPACE` but clients won't know/do that.

### Describe the solution you'd like

When NVDA is on a link with proper Link attributes - just allow Enter to actually trigger the click event of the link. As said I cannot imagine a single situation why enter should do something else or what other things a user might expect to happen when a user clicks `ENTER`

### Describe alternatives you've considered

Maybe listen for certain attributes on the link that developers can use?

### Additional context

This mostly happens in Chrome ( tested with 90.0.4430.212) Firefox mostly works. 

Edit: I know there are many other issues about that, but all of them are closed and/or years old so I thought I could create a new one

**Comment by Anubarak** [comment]
Turns out the issue was due to icons in the link. NVDA focused those "before" tags somehow and not the link.
We build the font again with special characters as content instead of normal symbols and now everything works
